src/components/MultiEmailInput/index.jsx

```diff
@@ -292,6 +295,10 @@ MultiEmailInput.propTypes = {
    * To specify whether a new email option can be created or not.
    */
   isCreateable: PropTypes.bool,
+  /**
+   * To specify whether the input field should always be expanded or not.
+   */
+  shouldAlwaysExpanded: PropTypes.bool,
```

Comment:
I think we should prefix with `is` since that's the standard we follow for naming boolean props/variables. Also, `is` sounds better than `should` over here.

```suggestion
  /**
   * To specify whether the input field should always be shown in an expanded state or not.
   */
  isAlwaysExpanded: PropTypes.bool,
```